runsc/cgroup/cgroup.go

```diff
@@ -212,9 +213,56 @@ func loadPathsHelper(cgroup io.Reader) (map[string]string, error) {
 	}
 	defer mountinfo.Close()
 
+	if libcontainercgroups.IsCgroup2UnifiedMode() {
+		return loadPathsHelperV2WithMountinfo(cgroup, mountinfo)
+	}
+
 	return loadPathsHelperWithMountinfo(cgroup, mountinfo)
 }
 
+func loadPathsHelperV2WithMountinfo(cgroup, mountinfo io.Reader) (map[string]string, error) {
```

Comment:
I _think_ that if cgroupns is used, there is no need to consult mountinfo -- the paths from /proc/<PID>/cgroup are valid as is even in case of nested containers (and are relative to /sys/fs/cgroup which should be hardcoded for v2).